server/src/games/Games.TyrantsOfTheUnderdark/TyrantsOfTheUnderdarkScoringStrategy.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.Text.Json;
+using BGS.ApplicationCore.Games.CalculateScore;
+
+namespace BGS.Games.TyrantsOfTheUnderdark;
+
+internal class TyrantsOfTheUnderdarkScoringStrategy : IGameScoringStrategy
+{
+    public int Calculate(JsonElement scoringData)
+    {
+        var model = scoringData.Deserialize<TyrantsOfTheUnderdarkScoringModel>();
+
+        return model.TrophyHall +
+            model.Deck +
+            model.InnerCircleDeck +
+            model.Tokens +
+            model.ControlSites +
+            model.TotalControlSites;
```

Comment:
I suggest the "micro" simplification of scoring on the user side and the general idea of the project to make scoring as simple as possible, make the multiplication here by 2.
After it add tests were:
- multiply by 0;
- multiply by 1;
- multiply by 2.